.github/workflows/deploy.yml

```diff
@@ -0,0 +1,38 @@
+# yamllint --format github .github/workflows/test.yml
+---
+name: Deploy master and release versions
+
+# We deploy on master and release versions, regardless of if the commit is
+# documentation-only or not.
+on:
+  push:
+    tags: '[0-9]+.[0-9]+.[0-9]+'
```

Comment:
Not sure if it matters to escape the dot (so it's not a wildcard match) or not

Reply:
sgtm